File: tessoku/A02 - Linear Search/A02.js
const main = (input) => {
    const lines = input.trim().split('\n');
    const [n, x] = lines[0].split(' ').map(Number);
    const numsA = lines[1].split(' ').map(Number);

    const includeNum = (nums, num) => {
        if (numsA.includes(num)) {
          console.log('Yes');
        } else {
          console.log('No');
        }
    }

    includeNum(numsA, x);
}

main(require('fs').readFileSync('/dev/stdin', 'utf8'));

File: tessoku/A02 - Linear Search/A02.cs
using System;
using System.Linq;

class Program
{
    static void Main()
    {
        string[] lines = Console.ReadLine().Trim().Split();
        int n = int.Parse(lines[0]);
        int x = int.Parse(lines[1]);

        int[] numsA = Console.ReadLine().Trim().Split().Select(int.Parse).ToArray();

        IncludeNum(numsA, x);
    }

    static void IncludeNum(int[] nums, int num)
    {
        if (nums.Contains(num))
        {
            Console.WriteLine("Yes");
        }
        else
        {
            Console.WriteLine("No");
        }
    }
}